src/Devices/WaterSensorData/WaterSensorVisuals/GatewayVisual.js

```diff
@@ -33,8 +33,7 @@ const getGatewayVisialData = async (gatewayNo, token, year) => {
 
 // Default Function
 const GatewayVisual = (props) => {
-  const gatewayNo = props.gatewayNo;
-  const [gatewayData, setGatewayData] = useState({});
+  const { gatewayNo, year } = props;
   const [loading, setLoading] = useState(false);
   const [volatageChartOptions, setVoltageChartOptions] = useState({
```

Comment:
Does this typo for `voltageChartOptions` have repercussions elsewhere? Would be nice to fix.